tests/config/indent_with_tabs-0.cfg

```diff
@@ -1 +1,2 @@
-indent_with_tabs                = 0
\ No newline at end of file
+indent_with_tabs      = 0
+sp_enum_before_assign = remove
```

Comment:
Could you either make a new config for that or rename this config file? 

Way back when I did the clean up of the test configs I spend some effort to name single option configs according to the pattern that you see in the config name (`<optionName>-<1stChar:optionValue>`).
I wanted to have that pattern in order to clearly see which option/value pairs are already having tests and which ones don't.

Reply:
Thanks.